platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -2110,7 +2109,6 @@
 			buildActionMask = 2147483647;
 			files = (
 				DA25D5C61CCDA06800607828 /* Root.strings in Resources */,
-				DA25D5C01CCD9F8400607828 /* Root.plist in Resources */,
```

Comment:
I’m confused by this change. We’re removing Root.plist from the Copy Files build phase, but it [remains the bundle’s Info plist](https://github.com/mapbox/mapbox-gl-native/blob/2031c2fb7175061005352f7ab66b1c2a93bac66e/platform/ios/ios.xcodeproj/project.pbxproj#L2720)?

Reply:
Which seems to be correct, no? It's what you get when you create a new framework from scratch. The plist is bundled in the framework but omitted from the Copy Files build phase.
 (https://developer.apple.com/library/content/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/FrameworkAnatomy.html)

Reply:
But the file in the resulting bundle is called Info.plist, not Root.plist, right? Does the Settings screen show the correct content after installing this bundle?

Reply:
🙈